app/uk/gov/hmrc/cataloguefrontend/leakdetection/LeakDetectionService.scala

```diff
@@ -71,17 +71,21 @@ class LeakDetectionService @Inject() (
         ).sortBy(_.unresolvedCount).reverse
       )
 
-  def repoSummaries(rule: Option[String], team: Option[String], includeWarnings: Boolean, includeExemptions: Boolean, includeViolations: Boolean)(implicit hc: HeaderCarrier): Future[(Seq[String], Seq[LeakDetectionRepositorySummary])] =
+  def repoSummaries(rule: Option[String], team: Option[String], includeWarnings: Boolean, includeExemptions: Boolean, includeViolations: Boolean, includeNoIssues: Boolean)(implicit hc: HeaderCarrier): Future[(Seq[String], Seq[LeakDetectionRepositorySummary])] =
     for {
       rules <- leakDetectionConnector.leakDetectionRules
       summaries <- leakDetectionConnector.leakDetectionRepoSummaries(rule, None, team)
-      filteredSummaries = filterSummaries(summaries, includeWarnings, includeExemptions, includeViolations)
-    } yield (rules.map(_.id), filteredSummaries.sortBy(_.repository))
+      filteredSummaries = filterSummaries(summaries, includeWarnings, includeExemptions, includeViolations, includeNoIssues)
+    } yield (rules.map(_.id), filteredSummaries.sortBy(_.repository.toLowerCase))
 
-  def branchSummaries(repo: String)(implicit hc: HeaderCarrier): Future[Seq[LeakDetectionBranchSummary]] =
+  def branchSummaries(repo: String, showAll: Boolean)(implicit hc: HeaderCarrier): Future[Seq[LeakDetectionBranchSummary]] =
```

Comment:
It isn't obvious what `showAll = false` would show - maybe call it `includeReposWithoutResults` or something?